### PR TITLE
fix: Add missing titles/descriptions to various schemas

### PR DESCRIPTION
**Pull request by ksbrar**

### CODE REVIEW

File: ludwig/schema/defaults/defaults.py [hunk 1]
```diff
@@ -37,9 +37,4 @@ class DefaultsConfig(schema_utils.BaseMarshmallowConfig):
 def get_defaults_jsonschema():
     """Returns a JSON schema structured to only require a `type` key and then conditionally apply a corresponding
     combiner's field constraints."""
-    defaults_schema = schema_utils.unload_jsonschema_from_marshmallow_class(DefaultsConfig)
```
Comment (ksbrar) [inline ludwig/schema/defaults/defaults.py:40]: @connor-mccorm sanity check: did this add anything?

Comment (connor-mccorm) [reply ludwig/schema/defaults/defaults.py:40]: Lol no, that's some dumb code right there!

**Review by connor-mccorm — APPROVED** [review]
LGTM!

File: ludwig/schema/combiners/utils.py [hunk 2]
```diff
@@ -19,7 +19,13 @@ def get_combiner_jsonschema():
     return {
         "type": "object",
         "properties": {
-            "type": {"type": "string", "enum": combiner_types, "default": "concat"},
+            "type": {
+                "type": "string",
+                "enum": combiner_types,
+                "default": "concat",
+                "title": "type",
```
Comment (martindavis) [inline ludwig/schema/combiners/utils.py:26]: If we're expecting this to be customer facing, should we capitalize `Type`?

Comment (ksbrar) [reply ludwig/schema/combiners/utils.py:26]: @martindavis all titles are lowercase in the schema - they get converted accordingly later

**Pull request merged**